feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -323,21 +323,26 @@ func TestZeroMaster(t *testing.T) {
 		highID:     inId0,
 		handle:     clientConnection(t, dut),
 		deviceID:   deviceId,
-		wantFail:   true,
-		wantStatus: 3,
+		wantWrite:  false,
+		wantRead:   true,
+		wantStatus: 0,
 	}
 	t.Run(test.desc, func(t *testing.T) {
 		resp, err := streamP4RTArb(&test)
-		if err == nil && test.wantFail {
-			t.Errorf("Zero ElectionID (0,0) is allowed: %v", err)
+		if err != nil {
 			removeClient(test.handle)
-		} else {
-			t.Logf("Zero ElectionID (0,0) connection failed as expected: %v", err)
+			t.Errorf("Zero ElectionID (0,0) is rejected by P4RT server: %v", err)
+			return
 		}
+		t.Logf("Zero ElectionID (0,0) connection success as expected")
 		// Validate status code
 		if resp != test.wantStatus {
 			t.Errorf("Incorrect status code received: want %d, got %d", test.wantStatus, resp)
 		}
+		// Validate the response for Read/Write for zero election ID
+		validateRWResp(t, &test)
```

Comment:
This should verify Get and Set of `ForwardingPipelineConfig`

Reply:
Updated the Testcase for two clients with unset electionId. Please check